Havtorn/Source/Engine/Graphics/RenderManager.cpp

```diff
@@ -25,6 +25,8 @@
 #include <DirectXTex/DirectXTex.h>
 #include <set>
 
+#include "Application/ImGuiManager.h"
```

Comment:
is this used anywhere in the RenderManager?

Reply:
It is not, leftover from earlier. Removed.